kbchat/kbchat_test.go

```diff
@@ -23,8 +25,19 @@ type Bots struct {
 	Charlie1 *OneshotOptions
 }
 
+type Team struct {
```

Comment:
not a huge fan of this name for a type that floating around in `kbchat`. also, it's not really a team. we have some unclear naming in this regard across a lot of the chat api interface. in my mind, a channel is the combination of a team name and a topic.